rotkehlchen/api/v1/encoding.py

```diff
@@ -838,6 +838,14 @@ class UserPremiumKeyRemoveSchema(Schema):
     name = fields.String(required=True)
 
 
+class UserPremiumSyncSchema(AsyncQueryArgumentSchema):
+    name = fields.String(required=True)
```

Comment:
Why do you need the name here? I would suggest removing it from the entire call. Just like all actions of the API they are for the logged in user. It should never ever be possible to perform an action for another user while they are not logged in. You even have a check in the actual function that the name user is the one who is logged in.

Reply:
I just copied the UserPremiumKeyRemove api since I nested the api under the `/users/:name/permium/sync`. I am open to any sugggestions on that.